test/sharness/t0100-name.sh

```diff
@@ -10,12 +10,14 @@ test_description="Test ipfs repo operations"
 
 test_init_ipfs
 
+for TTL_PARAMS in "" "--ttl=0s" "--ttl=1h"; do
+
 # test publishing a hash
 
-test_expect_success "'ipfs name publish' succeeds" '
+test_expect_success "'ipfs name publish${TTL_PARAMS:+ $TTL_PARAMS}' succeeds" '
 	PEERID=`ipfs id --format="<id>"` &&
 	test_check_peerid "${PEERID}" &&
-	ipfs name publish "/ipfs/$HASH_WELCOME_DOCS" >publish_out
+	ipfs name publish${TTL_PARAMS:+ $TTL_PARAMS} "/ipfs/$HASH_WELCOME_DOCS" >publish_out
```

Comment:
this syntax is strange to me, afaict its saying:

If `TTL_PARAMS` is null or unset, use the value of the variable named by `TTL_PARAMS`.

ref: http://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion